DataFormats/L1TParticleFlow/interface/HPSPFTau.h

```diff
@@ -0,0 +1,154 @@
+#ifndef DataFormats_L1TParticleFlow_HPSPFTau_H
+#define DataFormats_L1TParticleFlow_HPSPFTau_H
+
+#include "DataFormats/L1TParticleFlow/interface/PFCandidate.h"  // l1t::PFCandidate, l1t::PFCandidateRef, l1t::PFCandidateRefVector
+#include "DataFormats/L1TParticleFlow/interface/PFJet.h"    // l1t::PFJet, l1t::PFJetCollection, l1t::PFJetRef
+#include "DataFormats/Candidate/interface/LeafCandidate.h"  // reco::LeafCandidate
+#include "DataFormats/Candidate/interface/Particle.h"       // reco::Particle::LorentzVector
+#include "DataFormats/L1TCorrelator/interface/TkPrimaryVertex.h"
+
+#include <ostream>
+
+// forward declation needed in order to declare HPSPFTauBuilder class as friend (that has access to private data-members)
+class HPSPFTauBuilder;
+
```

Comment:
I don't actually see any `HPSPFTauBuilder` class in this PR.
Wouldn't it be better to remove the forward declaration and friend declaration for the time being ?
```suggestion
```

Reply:
It may well be! It is my understanding that the Tallinn group has a EDProducer that creates the HPSPFTaus.

Reply:
Usually we don't need `EDProducer`s to be `friend` of their products - but I can see why in some cases it could make sense.

Anyway, if it will be needed, I would rather add it only when the producer itself is added to CMSSW.